madgiraffestudios/src/App.scss

```diff
@@ -73,19 +73,16 @@ body {
     }
 }
 
-@media only screen and (max-width:768px) {
+@media only screen and (max-width:1024px) {
     .elevator-pitch { text-align: center;}
     .emphasized-text { font-size: 2rem;}
     .main-img { display: none;}
     .navbar-nav{ display: table;}
- 
+    .booking-btn { width: auto; float: none!important}
```

Comment:
why is this added to booking button?

Reply:
taking out 756 allowed me to add the button in the 1024...it changed nothing

Reply:
Addressed in next PR changed to 991px